src/Private/Extras/Set.elm

```diff
@@ -0,0 +1,36 @@
+module Private.Extras.Set where
+
+type alias Set = { start : Float, end : Float }
```

Comment:
Calling this `Set` can cause confusion with the built in `Set`. I recommend renaming this to `Interval` since it truly is an interval of numbers with a start and end point.